binderhub/app.py

```diff
@@ -646,6 +647,7 @@ def initialize(self, *args, **kwargs):
                 {'path': os.path.join(self.tornado_settings['static_path'], 'images')}),
             (r'/about', AboutHandler),
             (r'/health', HealthHandler, {'hub_url': self.hub_url}),
+            (r'/_config', ConfigHandler, {'repo_providers': self.repo_providers}),
```

Comment:
Actually, you don't need to pass repo_providers here. Because `repo_providers` config is passed to tornado app settings:

https://github.com/jupyterhub/binderhub/blob/207ae704752cde82f33fc771a3b58aa1ecc04b6d/binderhub/app.py#L590

So you can have this line as `(r'/_config', ConfigHandler),` and then in `ConfigHandler.generate_config` you can directly use `self.settings["repo_providers"]` to get value of `repo_providers` config and also you can delete `ConfigHandler.initialize` method.

Reply:
Thanks a lot for reviewing this @bitnik :sparkles: 
I'll rebase and address your feedback asap.